sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_gcp_stderr_events_v1/query.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+  -- example logger expected input: fxa-auth-server
+  SPLIT(jsonPayload.logger, "-")[OFFSET(1)] AS fxa_server,
+  * REPLACE (
+    (
+      SELECT AS STRUCT
+        jsonPayload.* REPLACE (
+          (
+            SELECT AS STRUCT
+              jsonPayload.fields.* EXCEPT (device_id, user_id),
+              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
+              TO_HEX(SHA256(jsonPayload.fields.device_id)) AS device_id
```

Comment:
Suggest using `REPLACE ()` instead so `user_id` and `device_id` aren't always pushed to the end.
```suggestion
            SELECT AS STRUCT
              jsonPayload.fields.* REPLACE (
                TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
                TO_HEX(SHA256(jsonPayload.fields.device_id)) AS device_id
              )
```
(ditto in `nonprod_fxa_gcp_stdout_events_v1`)

Reply:
The same still needs to be done in `nonprod_fxa_gcp_stdout_events_v1`.

Reply:
ok, I made the tweaks @sean-rose. I hope now everything looks in order.